src/JsonLd/Context.php

```diff
@@ -138,6 +138,9 @@ private function getContextType($name, array $data = [])
             case 'music_playlist':
                 return new ContextTypes\MusicPlaylist($data);
                 break;
+            case 'corporation':
```

Comment:
This is pretty messy. I just haven't had time to go back over it again. I plan on looking at cleaning this up in the coming months, if time allows for it.